core/src/main/java/org/apache/iceberg/DataFilesTable.java

```diff
@@ -122,7 +122,7 @@ public long targetSplitSize() {
     @Override
     public CloseableIterable<StructLike> rows() {
       return CloseableIterable.transform(
-          ManifestFiles.read(manifest, io).project(schema),
+          ManifestFiles.read(manifest, io, null, null).project(schema),
```

Comment:
Could we keep the original signatures as well, so we do not have to rewrite the method calls everywhere and add `null, null`? I think this could greatly reduce the changes needed in this patch